app/presenters/sufia/file_set_presenter.rb

```diff
@@ -13,6 +13,16 @@ def tweeter
       end
     end
 
+    def rights
+      return nil if solr_document.rights.nil?
+      solr_document.rights.first
+    end
+
+    def related_files
```

Comment:
I wonder if related files still makes sense in Sufia 7.0.0. Related files in Sufia < 7 was a way to say "all these GFs/FSs that were uploaded as a single Batch/UploadSet are probably related," but FSs are not collected in USs in Sufia 7. I'll make a strawperson proposal.

We should split the notion of related files:

 1. FileSets within a single Work should be related such that you can browse to other FSs in a Work without going back up/out to the Work show page.
 1. Works should be related by virtue of having been uploaded together in an UploadSet (same idea as GFs/Batch in Sufia < 7).

Does that make sense? (This also clearly goes above and beyond this ticket.)